### ISSUE TITLE
why not use pytorch ?

### ISSUE THREAD

**Issue by limitmhw** [andrewhou1/Shadow-Mask-Face-Relighting#6]

**Comment by andrewhou1** [comment]
At the time, I was much more familiar with Tensorflow than Pytorch. These days I tend to use both about equally, and both are still widely used in the computer vision community.
